### PR TITLE
Set up Sentry for Javascript

### PR DESCRIPTION
**Pull request by pranav7**

### CODE REVIEW

**Comment by pranav7** [comment]
Threw this error as an example: https://sentry.io/organizations/getcadet/issues/1601830484/?project=5192625&query=is%3Aunresolved

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 import { Application } from "stimulus"
 import { definitionsFromContext } from "stimulus/webpack-helpers"
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({dsn: "https://2234c89fb9f34a1aaae5a8910141202e@o374501.ingest.sentry.io/5192625"});
```
Comment (srikar0896) [inline app/javascript/packs/application.js:14]: I think we need not to track sentry in dev env. We can do this only for prod 
```suggestion
if (process.env.NODE_ENV === 'production') {
  Sentry.init({dsn: "https://2234c89fb9f34a1aaae5a8910141202e@o374501.ingest.sentry.io/5192625"});
}
```

**Comment by srikar0896** [comment]
Also, I couldn't check the error in sentry. Was the error message clear and readable with stack trace? If not we will have to implement an error boundary similar to https://gist.github.com/srikar0896/0e8aee7ac6110bea01afbe9c97d2a00f to capture the errors with the stack trace.

File: app/javascript/packs/application.js (hunk 1 above)
Comment (pranav7) [reply app/javascript/packs/application.js:14]: Makes sense! 👍

**Pull request merged**